### PR TITLE
uses musl libc setres{g,u}id

### PR DESCRIPTION
**Pull request by ekcasey**
* libc pushes uid/gid change to all threads
* pull in imgutil w/ bugfixes that ensure that docker client only needs a single socket connection

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,21 +1,21 @@
 GOCMD?=go
 GOFLAGS?=-mod=vendor
-GOENV=GOARCH=amd64 CGO_ENABLED=0
+GOENV=GOARCH=amd64 CGO_ENABLED=1
```
Comment (sclevine) [inline Makefile]: CGO should remain disabled for launcher.

File: priv/docker.go [hunk 2]
```diff
@@ -1,7 +1,8 @@
-package docker
+package priv
```
Comment (sclevine) [inline priv/docker.go]: Maybe use `internal` if this package is supposed to be private?

File: priv/docker.go [hunk 3]
```diff
@@ -46,11 +46,15 @@ func (*unclosableConn) Close() error {
 
 // connectSockOpt overrides the client dialer to always return the same socket connection
 func connectSockOpt(host *url.URL) (client.Opt, error) {
-	socketConn, err := net.Dial("unix", host.Path)
-	if err != nil {
-		return nil, err
+	conns := make(chan *unclosableConn, 5)
+	for i := 0; i < 5; i++ {
```
Comment (sclevine) [inline priv/docker.go]: `len(conns)` (or even better, `runtime.NumCPU`?)

File: priv/docker.go (hunk 2 above)
Comment (ekcasey) [reply priv/docker.go]: `priv` is for "privileges" not "private". Better naming suggestions welcome. I don't want to introduce `internal` w/o taking the time to be more thoughtful about which existing packages should be in it.

File: priv/docker.go (hunk 3 above)
Comment (ekcasey) [reply priv/docker.go]: outdated: moving in a different direction (preventing the need for multiple conns)

**Pull request merged**